darkcluster/src/main/java/com/linkedin/darkcluster/api/ResponseValidationMetricsHeader.java

```diff
@@ -0,0 +1,122 @@
+package com.linkedin.darkcluster.api;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.util.Map;
+import java.util.Objects;
+
+
+/**
+ * This represents the response validation metrics sent to dark clusters in the form of a request header
+ * This header stores two fields:
+ * 1) source: uniquely identifying the instance of the application running on a dispatcher. It will be a combination of hostname and instanceId (i000, i001...)
```

Comment:
The documentation in rest.li should be generic, because this is open source. So, for instance, instead of "It will be a combination of hostname and instanceId" say something like "For example, it can be a combination of the hostname and an instance identifier".